projects/swift-protobuf/fuzz.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import SwiftProtobuf
+
+@_cdecl("LLVMFuzzerTestOneInput")
+public func test(_ start: UnsafeRawPointer, _ count: Int) -> CInt {
+  let bytes = UnsafeRawBufferPointer(start: start, count: count)
+  var options = BinaryDecodingOptions()
+  options.messageDepthLimit = 256
+  do {
+    let _ = try ProtobufUnittest_NestedTestAllTypes.self.init(serializedData: Data(bytes),
+                                             options: options)
+  } catch BinaryDecodingError.messageDepthLimit {
+  } catch let e {
+    print("Unexpected error: \(e)");
```

Comment:
Not sure if you want "unexpected error", since nothing says the random binary payload will be a valid protobuf binary.  It just mean the parsing failed for the given reason (i.e. - how it was malformed).

Reply:
Indeed :-)
```
Unexpected error: malformedProtobuf
Unexpected error: truncated
```